docker/resource_docker_container_funcs.go

```diff
@@ -382,6 +383,18 @@ func resourceDockerContainerCreate(d *schema.ResourceData, meta interface{}) err
 		var mode int64
 		for _, upload := range v.(*schema.Set).List() {
 			content := upload.(map[string]interface{})["content"].(string)
+			contentBase64 := upload.(map[string]interface{})["content_base64"].(string)
+			if content == "" && contentBase64 == "" {
```

Comment:
It looks like you were trying to do `ConflictsWith` between these two attributes so we add the flipped version of this logic as well to make sure that a user doesn't submit both types of content. Something like 
```suggestion
			if content != "" && contentBase64 != "" {
			  return fmt.Errorf("Error with upload content: only one of `content` or `content_base64` can be specified")
			  }
```

Reply:
Also, I saw my comment was a suggestion to override the current implementation. I'll edit it and suggest doing both if statements for maximum validation coverage

Reply:
Yes, makes totally sense. I added a check for at least one of the `content` properties is set and that not both are set + test. Now it looks good

Reply:
Looks great!